src/content/reference/react/useSyncExternalStore.md

```diff
@@ -32,43 +32,43 @@ function TodosApp() {
 
-[See more examples below.](#usage)
+[Lihat contoh yang ada di bawah.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parameter {/*parameters*/}
 
-* `subscribe`: A function that takes a single `callback` argument and subscribes it to the store. When the store changes, it should invoke the provided `callback`. This will cause the component to re-render. The `subscribe` function should return a function that cleans up the subscription.
+* `subscribe`: Sebuah fungsi yang menerima sebuah argumen `callback` dan melanggankan itu ke tempat penyimpanan. Saat tempat penyimpanan berubah, fungsi ini akan memanggil `callback`. Ini akan menyebakan komponen di-*render* ulang. Fungsi `subscribe` harus mengembalikan fungsi yang membersihkan langganan tersebut.
 
-* `getSnapshot`: A function that returns a snapshot of the data in the store that's needed by the component. While the store has not changed, repeated calls to `getSnapshot` must return the same value. If the store changes and the returned value is different (as compared by [`Object.is`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is)), React re-renders the component.
+* `getSnapshot`: Sebuah fungsi yang mengembalikan sebuah *snapshot* dari data yang dibutuhkan komponen yang berada di tempat penyimpanan. Saat tempat penyimpanan masih belum berubah, pemanggilan `getSnapshot` berulang kali tetap harus mengembalikan nilai yang sama. Jika tempat penyimpanan berubah dan nilai kembalian juga berubah (saat dibandingkan dengan [`Object.is`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is)), React me-*render* ulang komponen tersebut.
 
-* **optional** `getServerSnapshot`: A function that returns the initial snapshot of the data in the store. It will be used only during server rendering and during hydration of server-rendered content on the client. The server snapshot must be the same between the client and the server, and is usually serialized and passed from the server to the client. If you omit this argument, rendering the component on the server will throw an error.
+* **opsional** `getServerSnapshot`: Sebuah fungsi yang mengembalikan *snapshot* awal dari data yang ada di tempat penyimpanan. *Snapshot* hanya akan digunakan saat proses *render* dilakukan oleh server dan saat hidrasi konten yang telah di-*render* oleh server ke klien. *Snapshot* server harus sama antara klien dan server, dan biasanya diserialisasi dan diserahkan dari server ke klien. Jika Anda mengabaikan argumen ini, proses *render* komponen di server akan memunculkan kesalahan.
 
-#### Returns {/*returns*/}
+#### Kembalian {/*returns*/}
 
-The current snapshot of the store which you can use in your rendering logic.
+*Snapshot* saat ini dari tempat penyimpanan yang dapat Anda gunakan di logika *render* Anda.
 
-#### Caveats {/*caveats*/}
+#### Perhatian {/*caveats*/}
```

Comment:
Pakai Caveat aja

```suggestion
#### Caveats {/*caveats*/}
```

Reply:
Ini sepertinya harus ditambahkan di Glosarium karena saya sebenarnya memutuskan untuk menerjemahkan ini dari salah satu halaman yang ada.
https://github.com/reactjs/id.react.dev/blob/362e8e83d05073a7f729b8a883a9b9b86b58246f/src/content/reference/react/useRef.md?plain=1#L48

Reply:
@samnap11 Yes, needs to be discussed and confirmed to the glossary: https://github.com/reactjs/id.react.dev/discussions/337

cc @mazipan @r17x @resir014 @zainfathoni

Reply:
@samnap11  sudah update untuk diskusi mengenain *ceveats*, lihat disini https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5768126. 

Boleh diubah yah!